### PR TITLE
Kahlia cities echoes basics + noodles, puppet, dice, comb

### PR DESCRIPTION
**Pull request by ultimatefiend**
This is kind of a mish mash of different features.

### CODE REVIEW

**Comment by ultimatefiend** [comment]
I'm not sure why so many changes are being committed?  Did I goof up my branch or something?

File: innovation.scss [hunk 1]
```diff
@@ -1231,6 +1231,14 @@
...
             right: 7px;
         }
+        &.top_right_icon {
+            top: 7px;
+            right: 7px;
+        }
+        &.top_center_icon {
+            top: 7px;
+            left: 72px;
+        }
```
Comment (lordmcfuzz) [inline innovation.scss:1241]: Probably should plop this onto the .L section too.

**Comment by micahstairs** [comment]
> I'm not sure why so many changes are being committed? Did I goof up my branch or something?

It's because I rebased it a few days ago (hoping you didn't have an active branch off of main-echoes). Don't worry I can fix this up :)

File: innovation.scss (hunk 1 above)
Comment (micahstairs) [reply innovation.scss]: Done.

**Review by micahstairs — APPROVED** [review]
This is great progress! Nice work.

**Pull request merged**